File: components/retail-store-domain/src/main/java/com/vmware/data/demo/retail/store/domain/BeaconRequest.java
package com.vmware.data.demo.retail.store.domain;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.beans.Transient;
import java.math.BigInteger;


@Data
@NoArgsConstructor
public class BeaconRequest {
	private CustomerIdentifier customerId;
	private String deviceId;
	private String uuid;
	private int major;
	private int minor;
	private int signalPower;
	
}

File: components/retail-market-domain/src/main/kotlin/com/vmware/data/retail/store/domain/BeaconRequest.kt
package com.vmware.data.retail.store.domain


data class BeaconRequest(
    var customerId: CustomerIdentifier? = null,
    var deviceId: String = "",
    var uuid: String = "",
    var major: Int = 0,
    var minor: Int = 0,
    var signalPower : Int = 0
)
